test/provisioner/provision_ibmcloud_initializer.go

```diff
@@ -194,7 +187,7 @@ func initVpcV1() error {
 }
 
 func initClustersAPI() error {
-	ibmcloudTrace("initClustersAPI()")
+	log.Info("initClustersAPI()")
```

Comment:
I think this could have the `Debug` level?

Reply:
Agree